homeworks/serhii.noskov_SerhiiNSN/GamesProject/games.html

```diff
@@ -108,6 +108,9 @@
             <section class="games">
                 <div class="games__container">
                     <div class="games__tittle">Study project "Games"</div>
+                    <div class="loading-overlay">
+                        <p>Loading...</p>
+                    </div>
```

Comment:
Hey, @SerhiiNSN, I have a question, why did you add a link to the card icon? You could add a link to it from `game_url` or remove it from html.
Also, the `div.card__icon` doesn't add any styles.
```html
                                 <a href="">
                                    <div class="card__icon">
                                        <img
                                            data-card-img
                                            src="img/Game/game-icon.jpg"
                                            alt=""
                                        >
                                    </div>
                                </a>
```

Reply:
you can fix it in the next hometask